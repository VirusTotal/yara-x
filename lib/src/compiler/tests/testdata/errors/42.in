rule test {
  strings:
    $a = /a.*b.*?c/
  condition:
    $a
}