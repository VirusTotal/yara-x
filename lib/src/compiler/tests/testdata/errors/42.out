error: mixing greedy and non-greedy quantifiers in regular expression
 --> line:3:15
  |
3 |     $a = /a.*b.*?c/
  |               ^^^ this is non-greedy
  |            ^^ this is greedy
  |